corehq/apps/hqadmin/reports.py

```diff
@@ -1216,8 +1216,8 @@ def get_data():
             rows.update({domain_name: [domain_name] + [0] * len(versions)})
 
         for data in get_data():
-            if data.commcare_version in versions:
-                row = rows.get(data.domain)
+            row = rows.get(data.domain, None)
+            if row and data.commcare_version in versions:
```

Comment:
I think it would be better to filter the data at the query level. And I also think it should be a global filter like these: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/forms.py#L13-L15